> updated the obstacle detection code

peacekurella opened the pull request:
Used iterative end point detection instead of RANSAC for improved speed and stability

peacekurella committed:
> updated the obstacle detection code

Yashasvi-Sriram commented:
The master is updated. Pull the master and resolve conflicts.
Please donot remove the RANSAC extraction code if that is what you are doing.

prashanth-flash commented:
Ransac is not required anymore, it only adds to the confusion

On Thu, Apr 30, 2020, 11:41 PM Yashasvi Sriram Patkuri <
notifications@github.com> wrote:

> The master is updated. Pull the master and resolve conflicts.
> Please donot remove the RANSAC extraction code if that is what you are
> doing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Yashasvi-Sriram/sixth-sense/pull/2#issuecomment-622245072>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD42V4Q6MVOV3NCJNL2G5YLRPJHHXANCNFSM4MW3GCTQ>
> .
>

peacekurella committed:
> bug fix

Yashasvi-Sriram commented:
Let it be there, I am working on it.
There can be multiple implementations of extraction logic.
Put yours in a separate place.
Besides if there is an improvement it can be compared to RANSAC based approach as a baseline.

peacekurella committed:
> Added class LandmarkObstacleExtractionLogicIEP
Merge branch 'master' of https://github.com/Yashasvi-Sriram/sixth-sense into feature/ITEP

peacekurella committed:
> indentation fixes

peacekurella committed:
> indentation fixes
> removed unsused vals
